### ISSUE TITLE
crash on startup

### ISSUE THREAD

**Issue by johndoe71rus** [Wargus/stratagus#580]
build master from git

[build_log.txt](https://github.com/Wargus/stratagus/files/13460552/build_log.txt)

failed at start 
[stratagus_run.txt](https://github.com/Wargus/stratagus/files/13460561/stratagus_run.txt)

lubuntu 22.04

**Comment by Jarod42** [comment]
It seems from the log you don't provide datadir path:

> LoadCcl: Maybe you need to specify another gamepath with '-d /path/to/datadir'

Startagus is "just" an engine and need game data. Have you installed wargus or another game data?

**Comment by johndoe71rus** [comment]
yes, I have the extracted data. I tried this set on 2 other computers. it works there, even appImage and data in wargus-x86_64.AppImage.home
![screen-2023-11-25-18-38-42](https://github.com/Wargus/stratagus/assets/7861172/5ed85ec8-3183-455d-a206-9506e6b637a1)

**Comment by johndoe71rus** [comment]
I definitely don't understand what the problem is. I'm confused.
Built stratagus
[stratagus_build.txt](https://github.com/Wargus/stratagus/files/13533491/stratagus_build.txt)
built wargus
[wargus_build.txt](https://github.com/Wargus/stratagus/files/13533492/wargus_build.txt)

works on the system where the build took place.
[wargus_run_at_buildpc.txt](https://github.com/Wargus/stratagus/files/13533494/wargus_run_at_buildpc.txt)

I'm running it on a test system, and there's an error.
johndoe@test:~/game/wargus$  ./wargus
Warning: Missing charsets in String to FontSet conversion

On the test system, appimage works with the same extracted data.
[wargus_run_at_testpc.txt](https://github.com/Wargus/stratagus/files/13533499/wargus_run_at_testpc.txt)

and yes, Stratagus not run as is without wargus or stargus

**Comment by MoriGM** [comment]
What Os is the Test system running on?
What Os is the Build system running on?
Are both just Lubuntu?
